dpnp/random/dpnp_iface_random.py

```diff
@@ -1289,7 +1289,7 @@ def sample(size):
     return call_origin(numpy.random.sample, size)
 
 
-def shuffle(x):
+def shuffle(x1):
```

Comment:
why `x1`?

Reply:
Because one letter variable is worse solution

Reply:
I see. It is just little bit confuse. I just thought `1` means `1` dimension (1 dimensional array) or something else.
But this is not critical. It is OK.